index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8"></meta>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0″></meta>
+  <title>Palindrome Checker</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      text-align: center;
+    }
+  </style>
+</head>
```

Comment:
- Your code has issues in here solve them:
![image](https://github.com/S-BLH/palindrome-checker/assets/96838030/1ee40f94-1d28-4ae5-bb2c-9cc2995df399)